fixtures/analysis_statuses.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "ANALYSIS_STATUS_ID": 1,
+    "ANALYSIS_STATUS": "scheduled"
+  },
+  {
+    "ANALYSIS_STATUS_ID": 2,
+    "ANALYSIS_STATUS": "running"
+  },
+  {
+    "ANALYSIS_STATUS_ID": 3,
+    "ANALYSIS_STATUS": "completed"
+  },
+  {
+    "ANALYSIS_STATUS_ID": 4,
+    "ANALYSIS_STATUS": "failed"
+  },
+  {
+    "ANALYSIS_STATUS_ID": 5,
+    "ANALYSIS_STATUS": "suppressed"
```

Comment:
Didn't we removed this with `is_suppressed`?

Reply:
> Didn't we removed this with `is_suppressed`?

That row is still present in EMG REL DB – possibly nothing fk's to it, but the goal here is to populate an empty DB to match that.